docs/serverless/06-01-function-cr.md

```diff
@@ -75,57 +75,62 @@ spec:
+| `ConfigMapCreated`               | `ConfigurationReady` | A new ConfigMap was created based on the Function CR definition.                                                          |
+| `ConfigMapUpdated`               | `ConfigurationReady` | The existing ConfigMap was updated after changes in the Function CR name, its source code or dependencies.                |
+| `ConfigMapError`                 | `ConfigurationReady` | The ConfigMap could not be created or updated due to an error.                                                            |
+| `JobFailed`                      | `BuildReady`         | The image with the Function's configuration could not be created due to an error.                                         |
+| `JobCreated`                     | `BuildReady`         | The Kubernetes Job resource that builds the Function image was created.                                                   |
+| `JobRunning`                     | `BuildReady`         | The Job is in progress.                                                                                                   |
+| `JobsDeleted`                    | `BuildReady`         | Previous Jobs responsible for building Function images were deleted.                                                      |
+| `JobFinished`                    | `BuildReady`         | The Job was finished and the Function's image was uploaded to the Docker Registry.                                        |
+| `DeploymentCreated`              | `Running`            | A new Deployment referencing the Function's image was created.                                                            |
+| `DeploymentUpdated`              | `Running`            | The existing Deployment was updated after such changes as the Function's image, scaling parameters, variables, or labels. |
+| `DeploymentFailed`               | `Running`            | The Function's Pod crashed or could not start due to an error.                                                            |
+| `DeploymentWaiting`              | `Running`            | The Function was deployed and is waiting for Deployment's readiness.                                                      |
+| `DeploymentReady`                | `Running`            | The Function was deployed and ready to serve.                                                                             |
+| `ServiceCreated`                 | `Running`            | A new Service referencing the Function's Deployment was created.                                                          |
+| `ServiceUpdated`                 | `Running`            | The existing Service was updated after needed changes.                                                                    |
+| `ServiceError`                   | `Running`            | There are some problems with Kubernetes Service.                                                                         |
```

Comment:
Yeah, that one is quite vague. So there is no particular reason but a given (human-readable) error pops up? Maybe sth like this:

```
The Kubernetes Service could not be created or updated due to an error.                                                          
```

Reply:
Same answer as with HPA, I'll add some example as to what can go wrong with them